docs/usage_guide.rst

```diff
@@ -860,6 +860,29 @@ After following these steps your balances in the dashboard will be updated inclu
    :alt: Loopring balances in the UI
    :align: center
 
+Monerium
+===========
+
+So long as you provide monerium credentials, all monerium transactions in mainnet, polygon and gnosis chain will be decorated with the bank data (or chain bridging), data you can also see in monerium app.
+
+1. For bank transfers from/to your address you will be able to see the destination/source IBAN along with the memo of the transfer.
+2. For automatic EURe bridging between EVM chains you will see amounts, from/to EVM chains along with from/to address.
+
+.. image:: images/monerium_transactions.png
+   :alt: Monerium decorated transactions
+   :align: center
+
+But there are two important pieces of information to keep in mind here.
```

Comment:
Keep only 1 piece of information here. Since this is general usage guide. Not release notes. So 2nd thing is not needed.